server/routes/initialAppointment/handlers/hardStop/initialMeetingPlace.ts

```diff
@@ -23,6 +23,10 @@ export default class InitialMeetingPlaceRoutes {
     await this.licenceService.updateAppointmentAddress(licenceId, req.body, user)
 
     flashInitialApptUpdatedMessage(req, licence, this.userType)
-    res.redirect(`/licence/hard-stop/create/id/${licenceId}/initial-meeting-contact`)
+    if (req.originalUrl?.includes('edit')) {
```

Comment:
rather than check originalUrl it might be better to check path as that will only include path and not any query parameters?